api/etcd/list_test.go

```diff
@@ -0,0 +1,94 @@
+package etcd
+
+import (
+	"context"
+	"testing"
+
+	"github.com/heetch/regula/api"
+	"github.com/heetch/regula/rule"
+	"github.com/stretchr/testify/require"
+)
+
+// List returns all rulesets rulesets or not depending on the query string.
```

Comment:
I think you need to rewrite this docstring, I'm not really sure what its trying to say.